app/js/components/orderEntry/styles.scss

```diff
@@ -362,3 +362,7 @@
     }
   }
 }
+
+.patient-header {
+  margin: 10px 0px;
+}
```

Comment:
@Rhotimee, please add an end of file line.

Reply:
done.